packages/expo-modules-core/src/createWebModule.ts

```diff
@@ -0,0 +1,6 @@
+import { NativeModule } from './web/CoreModule';
```

Comment:
This makes non-web platform import web-specific code

Reply:
I assumed that it would never be imported by a file that is not a `.web` platform specific one – if someone wants to run `createWebModule` on other platform – would you expect it to do nothing or wrap JS code with a JS only event emitter?

Reply:
`createWebModule` isn't really web-specific, in theory it could be used on other platforms with the same implementation. We just want to use `web` in its name to indicate that users should use it only on web.

The problem here is that you're also doing `export * from './createWebModule';` in the index file, so `web/CoreModule` would be included in the bundle for everyone.

Reply:
I'm not sure that is the case – docs state that the values do not become available in the current file with this export declaration, so I'd assume it is bundled only when you actually import it somewhere – should I check it?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#re-exporting_aggregating

Reply:
You're right – even an aggregate export causes the file to be always included in the bundle, even if not imported in client code :/